app/controllers/users_controller.rb

```diff
@@ -0,0 +1,52 @@
+class UsersController < ApplicationController
+
+  get '/signup' do
+    if logged_in?
+      redirect to '/tweets'
+    end
+    erb :'/users/create_user'
+  end
+
+  post '/signup' do
+    if logged_in?
+      redirect '/tweets'
+    elsif params.values.include?('')
```

Comment:
I think you could use [ActiveRecord validations](https://learn.co/tracks/full-stack-web-development/sinatra/activerecord/user-authentication-in-sinatra) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.